modules/ossm-security-mtls.adoc

```diff
@@ -23,10 +23,24 @@ kind: ServiceMeshControlPlane
 spec:
   version: v2.0
   security:
-    controlPlane:
+    dataPlane:
       mtls: true
 ----
 
+You can also set this in the {product-title} web console
+
+. Log in to the web console.
```

Comment:
Probably out of scope for this PR, but you guys should look at updating these sections to be more aligned with mod docs. There should be separate modules for all of these subsections, and any that are tasks should be styled as procedure modules (with the .Procedure heading, etc.).

Reply:
@bergerhoffer I hear you. I'll make a new work item for that. I think we have a lot of that kind of work coming up. 

I have knocked out the rest of your comments. Thanks for taking a look. I also wish to state for the record here that I feel sheepish about forgetting a period or two. Thanks for catching those. :) 

Anyway, ready for re-review.

Reply:
Awesome thank you! And no worries, it happens to the best of us. That's what peer review is for :) 

Taking another look now.

Reply:
https://issues.redhat.com/browse/OSSMDOC-278